<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
   <item>
       <widget class="QPushButton" name="resetButton">
           <property name="text">
               <string>Reset</string>
           </property>
       </widget>
   </item>

    <item>
     <widget class="QLineEdit" name="commandLineEdit"/>
    </item>
    <item>
     <widget class="QTextEdit" name="gameText"/>
    </item>
    <item>
     <widget class="QPushButton" name="restartButton">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Рестарт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <action name="actionSubmitCommand">
    <property name="text">
     <string>Введіть команду</string>
    </property>
   </action>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
